core/connectors/sdk/src/transforms/add_fields.rs

```diff
@@ -16,39 +16,31 @@
  * under the License.
  */
 
+use super::{Transform, TransformType};
+use crate::{DecodedMessage, Error, Payload, TopicMetadata};
 use serde::{Deserialize, Serialize};
 use simd_json::OwnedValue;
 use strum_macros::{Display, IntoStaticStr};
 
-use crate::{DecodedMessage, Error, Payload, TopicMetadata};
-
-use super::{Transform, TransformType};
-
-#[derive(Debug, Serialize, Deserialize)]
-pub struct AddFieldsConfig {
-    fields: Vec<Field>,
-}
-
-pub struct AddFields {
-    fields: Vec<Field>,
-}
-
+/// A field to be added to messages
 #[derive(Debug, Serialize, Deserialize)]
-struct Field {
-    key: String,
-    value: FieldValue,
+pub struct Field {
+    pub key: String,
+    pub value: FieldValue,
 }
 
+/// The value of a field, either static or computed at runtime
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "snake_case")]
-enum FieldValue {
```

Comment:
The `FieldValue` and `ComputedValue` are the same for both, `add_fields` and `update_fields` - can we make this shared & reusable?